eqs/src/routes.rs

```diff
@@ -171,6 +172,19 @@ pub async fn get_transaction_by_hash(
     }
 }
 
+/// Return a JSON expression with status 200 indicating the server
+/// is up and running. The JSON expression is simply,
+///    {"status": "available"}
+/// When the server is running but unable to process requests
+/// normally, a response with status 503 and payload {"status":
+/// "unavailable"} should be added.
+pub async fn healthcheck() -> Result<tide::Response, tide::Error> {
```

Comment:
Looks good for an initial version. Is there going to be support for any other status? EQS has a sort of implicit "catching up" state when it starts up, and it might be worth actually representing that somehow.

Reply:
Let's discuss this on Zulip.